manifests/resource/location.pp

```diff
@@ -217,6 +220,8 @@
   Optional[String] $proxy_http_version                 = undef,
   Optional[String] $proxy_set_body                     = undef,
   Optional[Enum['on', 'off']] $proxy_buffering         = undef,
+  Optional[String] $proxy_max_temp_file_size           = undef,
```

Comment:
Can you please use `Optional[String[1]]` here please?

Reply:
There is a size datatype which should be used in my opinion.

Reply:
good catch